dicom-archive/DICOM/Fields.pm

```diff
@@ -1,9 +1,62 @@
-# Definitions of fields of DICOM headers.
-# Andrew Crabb (ahc@jhu.edu), May 2002.
-# $Id: Fields.pm 4 2007-12-11 20:21:51Z jharlap $
-
 package DICOM::Fields;
 
+=pod
+
+=head1 NAME
+
+DICOM::Fields -- Definitions of fields of DICOM headers.
+
+=head1 SYNOPSIS
+
+  use DICOM::Element;
+  use DICOM::Fields;	# Standard header definitions.
+  use DICOM::Private;	# Private or custom definitions.
+
+  # Initialize dictionary.
+  # Read the contents of the DICOM dictionary into a hash by group and element.
+  # dicom_private is read after dicom_fields, so overrides common fields.
+  BEGIN {
+    foreach my $line (@dicom_fields, @dicom_private) {
+      next if ($line =~ /^\#/);
+      my ($group, $elem, $code, $numa, $name) = split(/\s+/, $line);
+      my @lst = ($code, $name);
+      $dict{$group}{$elem} = [@lst];
+    }
+  }
+
+
+=head1 DESCRIPTION
+
+Definitions of fields of DICOM headers. Not for medical use. This file is
```

Comment:
Not for medical use? What do you mean by that?

Reply:
I did not write this. It was the available description before formatting so I did not edit it. Not sure it is really relevant so I'll remove it.